tests/integration/pilot/mirror_test.go

```diff
@@ -155,9 +156,10 @@ func runMirrorTest(t *testing.T, options mirrorTestOptions) {
 										if expected == nil {
 											expected = apps.C
 										}
-
-										return verifyTrafficMirror(apps.B, expected, c, testID)
-									}, echo.DefaultCallRetryOptions()...)
+										return retry.UntilSuccess(func() error {
+											return verifyTrafficMirror(apps.B, expected, c, testID)
+										}, []retry.Option{retry.Timeout(20 * time.Second), retry.BackoffDelay(10 * time.Millisecond), retry.Converge(2)}...)
```

Comment:
I would consider it broken if the logs are slow. In what environment are they slow?